code/modules/clothing/clothing.dm

```diff
@@ -38,7 +38,17 @@
 	// THESE OVERRIDE THE HIDEHAIR FLAGS
 	var/dynamic_hair_suffix = ""//head > mask for head hair
 	var/dynamic_fhair_suffix = ""//mask > head for facial hair
+	var/greyscale = FALSE
+	var/my_clothing_icon = 'icons/mob/uniform.dmi'

```

Comment:
This is only used for greyscale, mention that (other icons would use the ```alternate_worn_icon``` var anyway, so this is PURELY for greyscale)